app/screens/CarPool/CarPool.js

```diff
@@ -1,19 +1,29 @@
+/* eslint-disable react-hooks/exhaustive-deps */
 import React, {useEffect, useState} from 'react';
 import {View, Text} from 'react-native';
 import {ScrollView, TouchableOpacity} from 'react-native-gesture-handler';
+import {useDispatch, useSelector, useStore} from 'react-redux';
 
 import {carGenerator} from '../../helpers';
-import {WORKSHOPS, SCREEN_NAMES} from '../../constants';
+import {WORKSHOPS, SCREEN_NAMES, REDUCERWORKSHOPS} from '../../constants';
 import Car from '../../components/Car/Car';
-import Workshop from '../../components/Workshop/Workshop';
+import Workshop from '../../components/WorkShop/WorkShop';
 import styles from './styles';
 
 const Cars = ({navigation}) => {
-  const [cars, setCars] = useState([]);
+  const mechanic = useSelector(store => store.garage.mechanic);
+  const tyreShop = useSelector(store => store.garage.tyreShop);
+  const bodyShop = useSelector(store => store.garage.bodyShop);
+  const cars = useSelector(store => store.garage.home);
```

Comment:
not a mistake, but you could do this with destructuring and call useSelector only once